go.mod

```diff
@@ -26,27 +23,21 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	github.com/golangci/golangci-lint v1.18.0
+	github.com/golangci/golangci-lint v1.42.0
 	github.com/google/go-cmp v0.5.6
 	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/google/renameio v0.1.0
-	github.com/gostaticanalysis/analysisutil v0.0.3 // indirect
-	github.com/hashicorp/golang-lru v0.5.3 // indirect
-	github.com/huandu/xstrings v1.2.0 // indirect
 	github.com/imdario/mergo v0.3.12
-	github.com/mattn/go-isatty v0.0.12 // indirect
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/openshift/api v0.0.0-20210730095913-85e1d547cdee
 	github.com/openshift/client-go v0.0.0-20210521082421-73d9475a9142
 	github.com/openshift/library-go v0.0.0-20210702104503-39570b4a2ae8
 	github.com/openshift/runtime-utils v0.0.0-20210722191527-8b8348d80d1d
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.7.1
-	github.com/securego/gosec v0.0.0-20191002120514-e680875ea14d
```

Comment:
We can also remove `gosec` from replace section as well (https://github.com/openshift/machine-config-operator/blob/master/go.mod#L75)

Reply:
good catch, thanks! Updated